test-files/MGCPv3/AAL015-output.osm

```diff
@@ -1,50 +1,49 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All this OGR related test output changed due to duplicated nodes being in the original test files. I painstakingly checked each one both visually and in meld. If I missed any major changes that shouldn't have been introduced, let me know.